database/migrations/2025_09_04_000002_create_matchmaking_pools_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('matchmaking_pools', function (Blueprint $table) {
+            $table->increments('id');
+            $table->smallInteger('ruleset_id');
+            $table->string('name');
+            $table->boolean('active');
+            $table->timestamps();
```

Comment:
the columns generated by this has no default value so whatever doing the insert will have to set them explicitly